src/components/AddResearcher/index.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import {
+  Button,
+  Col,
+  DatePicker,
+  Form,
+  Input,
+  Row,
+  Select }
+from 'antd';
+import moment from 'moment'
+import { statesAb } from '../../assets/data/states'
+import { contactMethods } from './constants'
+
+
+class AddResearcherForm extends React.Component {
+  handleSubmit = e => {
+    e.preventDefault();
+
+    this.props.form.validateFieldsAndScroll((err, fieldsValue) => {
+      if (err) return
+
+      // Format date value before submit.
+      const values = {
+        ...fieldsValue,
+        firstContact: fieldsValue.firstContact ? fieldsValue.firstContact.format('YYYY-MM-DD') : '',
+        volConfirmation: fieldsValue.volConfirmation ? fieldsValue.volConfirmation.format('YYYY-MM-DD') : ''
+      }
+      this.props.saveUnregisteredVol(values)
+      this.props.form.resetFields()
+    })
+  }
+
+  setValidationRules = (required=false, initialValue='', message='This field is required') => {
+    return {
+      initialValue,
+      rules: [{
+        required,
+        message
+      }]
+    }
+  }
+
+  parseStates = () => {
+    return Object.entries(statesAb).map(state => {
```

Comment:
No need to change here but I will often use `usState` instead of `state` as a variable name to make it clear it's not react state